stock_picking_package_preparation/readme/USAGE.rst

```diff
@@ -0,0 +1,14 @@
+To use this module, you need to:
+
+ * Go to Warehouse > Package Preparation
+
+From there, you can create a new preparation.
+Choose a partner then the selection of pickings you want in the package.
+You can choose a logistic unit or a packaging, they will be set on the
```

Comment:
How can I choose a logistic unit on the pack?

Reply:
Logistic units existed in v8, as far as I know they were deleted in v10 already, anyway I've deleted that line in https://github.com/OCA/stock-logistics-workflow/pull/448/commits/3b10189d10ac3fa3e82aeeca990a61c6b566afbd